codespell_lib/data/dictionary.txt

```diff
@@ -12360,6 +12504,7 @@ rectange->rectangle
 rectanges->rectangles
 rectanglar->rectangular
 rectiinear->rectilinear
+rects->rect
```

Comment:
also programming slang for "rectangles"

Reply:
done

Reply:
This says it's only an abbreviation, remove it? https://www.collinsdictionary.com/dictionary/english/rect

Reply:
Removed